client/src/modules/cohort/index.jsx

```diff
@@ -124,6 +124,15 @@ class Cohort extends PureComponent {
     return cohortDetails && cohortDetails.isArchived;
   };
 
+  checkIfAuthorized = () => {
+    const {
+      currentUser: { id: userId },
+      cohortDetails: cohort
+    } = this.props;
+
+    return cohort && cohort.adminIds && cohort.adminIds.includes(userId);
```

Comment:
data leak! adminIds should not be sent to client - cohots and lists right should be checked on backend and some flags `true/false` should be set on data which is send to client